client/src/components/MediaItemViews.tsx

```diff
@@ -0,0 +1,511 @@
+import { isSameDay, parseISO } from 'date-fns';
+import { minBy } from 'lodash';
+import { FC, Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+import { Plural, Trans } from '@lingui/macro';
+
+import { Button } from './Button';
+import { RelativeTime } from './Date';

```

Comment:
```suggestion
import { RelativeTime } from './date';
```
Or rename to `date.tsx` to `Date.tsx`

Reply:
I have renamed `date.tsx` to `Date.tsx`